### ISSUE TITLE
Add documentation as a static site on GitHub pages

### ISSUE THREAD

**Issue by bdsoha** [petereon/expycted#12]
Add documentation as a static site on GitHub pages.

- [ ] Add CI deployment.
- [ ] Create `docs` directory or a separate repo for the `.md` files

**Comment by petereon** [comment]
Are you comfortable if I bump the version to `1.0` after this and #10 get closed?

**Comment by bdsoha** [comment]
@petereon Comfortable? It is your project....

IMO, I would wait until some more *core* refactoring is complete, but it is up to you.

For example, I added a test utility that very much simplifies `pytest.mark.parametrize` conditions:
https://github.com/petereon/expycted/blob/a0797349d4806c9ac606dd45d0748ec5c6379a6e/test/expect_to_test_suite.py#L401-L415

I suggest that a separate Issue/PR be opened designated to that specific refactor.

**Comment by petereon** [comment]
Yes, I am asking because I am not sure if you have any potentially breaking changes on your mind going ahead. We don't need to be particularly mindful of these until `1.0`. 

It sounds reasonable to wait until the feature you suggest here.

**Comment by bdsoha** [comment]
I am not planning any breaking changes, just refactoring...
